Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -367,63 +317,49 @@ u32 Read_Opcode(u32 _Address)
 	return PowerPC::ppcState.iCache.ReadInstruction(_Address);
 }
 
+#ifdef ENABLE_MEM_CHECK
+#define MEMCHECK(write, size)\
```

Comment:
Don't assume the existence of locals in your macro. Instead, pass _Address and _var as arguments.